js/src/Ice/ConnectionI.js

```diff
@@ -856,7 +851,7 @@ export class ConnectionI {
         const info = this._transceiver.getInfo();
         for (let p = info; p !== null; p = p.underlying) {
             p.adapterName = this._adapter !== null ? this._adapter.getName() : "";
-            p.incoming = this._incoming;
+            p.incoming = false;
```

Comment:
Why do we need this incoming field, which I assume means whether the connection is incoming or outgoing?

Reply:
This was part of the generated ConnectionInfo public API. I guess we can remove it for JavaScript in 3.8